sql/core/pom.xml

```diff
@@ -146,6 +146,11 @@
       <artifactId>parquet-avro</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
do you need this? mockito is already present in test scope for spark/core/pom.xml which is inherited by spark/sql/core test

Reply:
I guess not. My IDE reported that I did.